client/src/Pages/PotentialTerms.js

```diff
@@ -13,7 +13,21 @@ export default class PotentialTerms extends Component {
   componentDidMount () {
```

Comment:
@saragw6 how do you feel about this

Reply:
I like this. So then we would create `rejectCb` for potential definitions as well so we can just call `entry.rejectCb()`, right?

Aside from that it might be easier to read the fetch call if the mapping function was refactored out into a helper fn